docs/index.md

```diff
@@ -0,0 +1,23 @@
+# Beetbox - Drupal development on Vagrant [![Circle CI](https://circleci.com/gh/drupalmel/beetbox.svg?style=svg)](https://circleci.com/gh/drupalmel/beetbox)
+
+## Overview
+
+Beetbox is a Vagrant configuration that provides your team with a versatile development environment for Drupal projects using Virtualbox for VMs (virtual machines).
+
+## Why use Beetbox?
+
+There are a number of Drupal-focussed Vagrant projects.
+
+* It is leaner than most other projects.
```

Comment:
Can we please add 2 more points here (please fix wording):
* Pre-provisioned box - takes less time to kickstart the developement
* Allows granular configuration